styles/weather_widget.css

```diff
@@ -0,0 +1,164 @@
+.clearfix:after {
+  content: "";
+  display: block;
+  clear: both;
+  visibility: hidden;
+  height: 0;
+}
+
+#weather-widget {
+  -webkit-box-sizing: border-box;
+  box-sizing: border-box;
+  width: 696px;
+  height: 435px;
+  font-family: Lato, Arial, sans-serif;
+  font-weight: 300;
+  line-height: 1;
+  color: white;
+  background: url("../img/bg_paris.jpg") #D8DEE2;
+}
+
+#weather-widget * {
+  -webkit-box-sizing: border-box;
+  box-sizing: border-box;
+}
+
+#weather-widget #main-part {
+  position: relative;
+  height: 327px;
+  padding: 68px 60px 0;
+}
+
+/* ======================================== */
+/*  Current day                             */
+/* ======================================== */
```

Comment:
I'm not sure you need this commented description.

Reply:
I've deleted some comments, but some -- left. I don't have much experience so comment really help me to read the code and work.
[Fixed](https://github.com/js-training-sep-2020/hw-1/pull/5/commits/7d06cedbf2742076d69e6b5d398e295714e2fa3c)

Reply:
Ok, I see